pom.xml

```diff
@@ -150,6 +152,34 @@
                     </dependency>
                 </dependencies>
             </plugin>
+            <plugin>
+                <groupId>com.dkanejs.maven.plugins</groupId>
```

Comment:
is that a well adopted plugin?

Reply:
Seems to be, 77 stars and 40 forks. A search across Github has it in use by both Apache and InfluxDB.

The project is archived now but it was the only one I found that supported Java 8, the others all had a minimum dependency on Java 11+